neptune/man/neptune_add.Rd

```diff
@@ -0,0 +1,42 @@
+\name{neptune_add}
+\alias{neptune_add}
+\title{
+Adds the provided tag or tags to the run's tags.
+}
+\description{
+Adds the provided tag or tags to the run's tags.
+}
+\usage{
+neptune_add(x, value)
+}
+\arguments{
+  \item{x}{Neptune handler, for which we want to add tags eg. run['Sys/tags']}
+  \item{value}{string or list of strings - Tag or tags to be added. }
+}
+\value{
+None
+}
+\author{
+Authors:
+\itemize{
+  \item Mateusz Dominiak
+  \item Maintainer: Jakub Czakon \email{jakub.czakon@neptune.ai}
+}
+}
+\seealso{
+Useful links:
+\itemize{
+  \item \url{https://neptune.ai/}
+  \item Documentation \url{https://docs.neptune.ai/}
+  \item Report bugs at \url{https://github.com/neptune-ai/neptune-r/issues}
+}
+}
+\examples{
+\dontrun{
+# We are using api token for an anonymous user neptuner. For your projects use your private token.
+run <- neptune_init(api_token = 'ANONYMOUS',
+                    project = 'shared/r-integration')
+neptune_add(run['Sys/tags'], 'some_tag')
```

Comment:
How could I initialize an empty `StringSet` in R?

Reply:
hmm.. that's a good question, typically one could create a empty vector like c() which then 'd get converted to list() on python's side

Reply:
An empty vector will land in `is.null` so it will be same as deleting an object `... <- NULL` (didn't checked only a way of thinking)

Reply:
hmm, true, okay, I'll add StringSet method which just returns empty StringSet maybe ... ?

Reply:
Leave it for now. I'll think about it :wink:

Reply:
Ohh. Sorry, I'm so stupid. On the Python side there is no way to create StringSet currently. Only `run['sys/tags']` is precreated. Closing